react-mockup/src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
```suggestion
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <meta name="description" content="This is an example of a meta description. This will often show up in search results.">
```
Another option to up your meta game. These articles are a good follow up:
- https://support.google.com/webmasters/answer/79812?hl=en
- https://moz.com/blog/seo-meta-tags